src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -45,12 +45,23 @@ case class Bagger(
     } else {
       trainingData.size
     }
+    val minBags = Math.log(1 - Math.pow(2, -1.0 / trainingData.size)) / Math.log((Math.E - 1) / Math.E)
```

Comment:
Can you add an inline explanation for this?

Reply:
Added